app/models/game.rb

```diff
@@ -4,4 +4,89 @@ class Game < ApplicationRecord
   belongs_to :player_1, class_name: 'Player', optional: true
   belongs_to :player_2, class_name: 'Player', optional: true
   belongs_to :winning_player, class_name: 'Player', optional: true
+  # Is called after  Base.save on new objects that have not been saved yet (no record exists).
+  after_create :populate_game!
```

Comment:
Since this is called in the controller. We don't need it in an `after_create`

I find it works better if we don't use `after_create`. Because it can be confusing when writing tests. Each time you create a game, it automatically has all pieces. Which isn't always what you want.